src/test/java/com/imgix/test/TestAll.java

```diff
@@ -63,40 +69,54 @@ public void testSetUseHttpsTrue() {
     @Test
     public void testHelperBuildAbsolutePath() {
         URLHelper uh = new URLHelper("securejackangers.imgix.net", "/example/chester.png", "http");
-        assertEquals(uh.getURL(), "http://securejackangers.imgix.net/example/chester.png");
+        assertEquals("http://securejackangers.imgix.net/example/chester.png", uh.getURL());
     }
 
     @Test
     public void testHelperBuildRelativePath() {
         URLHelper uh = new URLHelper("securejackangers.imgix.net", "example/chester.png", "http");
-        assertEquals(uh.getURL(), "http://securejackangers.imgix.net/example/chester.png");
+        assertEquals("http://securejackangers.imgix.net/example/chester.png", uh.getURL());
     }
 
     @Test
     public void testHelperBuildNestedPath() {
-        URLHelper uh = new URLHelper("securejackangers.imgix.net", "http://www.somedomain.com/example/chester.png", "http");
-        assertEquals(uh.getURL(), "http://securejackangers.imgix.net/http%3A%2F%2Fwww.somedomain.com%2Fexample%2Fchester.png");
+        URLHelper uh = new URLHelper("securejackangers.imgix.net", "http://www.somedomain.com/example/chester.png",
+                "http");
+        assertEquals("http://securejackangers.imgix.net/http%3A%2F%2Fwww.somedomain.com%2Fexample%2Fchester.png",
+                uh.getURL());
     }
 
-    @Test
-    public void testHelperBuildAbsolutePathWithParams() {
-        URLHelper uh = new URLHelper("securejackangers.imgix.net", "/example/chester.png", "http");
-        uh.setParameter("w", 500);
-        assertEquals(uh.getURL(), "http://securejackangers.imgix.net/example/chester.png?w=500");
-    }
+    @RunWith(Parameterized.class)
+    public static class testHelperBuildPathWithParams {
```

Comment:
Changes `testHelperBuild<PathType>WithParams` tests into one parameterized test, as suggested by sonar-lint. This will be useful when adding path encoding test cases moving forward. Also allows for more meaningful test error messages when debugging.